### DISCUSSION TITLE
Misc column letters meaning

### DISCUSSION THREAD

**Discussion by edison23** [rakshasa/rtorrent#1236] [Q&A] [answered]
Hi, I feel this surely is answered somewhere but I can't find it. I'm using rtorrent in terminal on Ubuntu and in one of the two available views, there's this last column named `Misc`. Some of the torrents in the list have a letter there; now I see `T` and `I` in my list (I'm pretty sure I've seen other letters too).

![image](https://github.com/rakshasa/rtorrent/assets/1723800/72f290f8-0aaf-4481-9d70-953088462aca)

One torrent changed its letter from nothing to `I` after I moved its destination (it was seeding > closed > moved (in Bash outside rtorrent) > ctrl+O > insert the new path to which I moved the files > ctrl+S (attempt to start seeding again -- and the `I` flag appears).

What do these letters mean? What part of rtorrent docs did I miss?

Thank you and pardon my ignorance.

**Comment by kannibalox** [comment, marked as answer]
`I` means "ignored" (specifically, it's most likely being automatically set as such because rtorrent can't see the files at the new location). https://rtorrent-docs.readthedocs.io/en/latest/cmd-ref.html#term-d-ignore-commands has some more information on exactly what that menas.

`T` means "tied to a .torrent file". https://rtorrent-docs.readthedocs.io/en/latest/cmd-ref.html#term-d-tied-to-file has more information.

https://github.com/rakshasa/rtorrent/wiki/User-Guide#main-view-window is a good overview of what the different things in the TUI mean. The download list will look a little different in the compact view, but the information is essentially the same.

**Reply by edison23** [reply]
Thank you very much! ^__^
